app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -498,18 +498,22 @@ class CatalogueController @Inject() (
       }
     }
 
-  def allDefaultBranches(singleOwnership: Option[Boolean], archived: Option[Boolean]): Action[AnyContent] = {
+  def allDefaultBranches(singleOwnership: Boolean, archived: Boolean): Action[AnyContent] = {
```

Comment:
can we rename `archived` to `includeArchived`? otherwise you may expect `archived=true` would only return archived repositories.